_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -48116,14 +48116,15 @@
 /turf/closed/wall,
 /area/station/security/checkpoint/medical)
 "nKb" = (
-/obj/machinery/door/window/left/directional/west{
-	name = "Secure Art Exhibition";
-	req_access = list("library")
-	},
 /obj/effect/spawner/random/structure/table_fancy,
 /obj/structure/sign/painting/library_secure{
 	pixel_x = 32
 	},
+/obj/machinery/door/window/right{
+	dir = 8;
```

Comment:
```suggestion
/obj/machinery/door/window/right/directional/west{
```

Reply:
You'll want to use the prefab rather than rotating it in the editor. That should get the tests to pass.